### PR TITLE
Config Hotfix: Prevent per-game settings (ie. GameINI) being stored to the global user configuration.

### PR DESCRIPTION
**Pull request by JosJuice**
#2687 rebased from stable to master.

### CODE REVIEW

**Comment by phire** [comment]
oh god

File: Source/Core/Core/ConfigManager.cpp [hunk 1]
```diff
@@ -82,6 +82,13 @@ SConfig::~SConfig()
 
 void SConfig::SaveSettings()
 {
+	// TODO: The is a hotfix to prevent writing of temporary per-game settings
```
Comment (lioncash) [inline Source/Core/Core/ConfigManager.cpp]: This

**Comment by MaJoRoesch** [comment]
This appears to have caused a regression: https://bugs.dolphin-emu.org/issues/9197

**Comment by phire** [comment]
I've reverted this for now. Either we leave the original bug for 5.0, or we need to come up with a better fix

**Pull request merged**